include/sdformat_urdf/sdformat_urdf.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include <sdf/sdf.hh>
+#include <urdf_world/types.h>
+#include <urdf_model/types.h>
+
+namespace sdformat_urdf {
+/// \brief Parse an SDFormat XML string and return URDF C++ structures
+urdf::ModelInterfaceSharedPtr
+parse(const std::string & data, sdf::Errors & errors);
```

Comment:
Dumb question: Do these need any visibility macros, given that these look like public API?

Reply:
Added visibility macros in 3c0a28f